### PR TITLE
Menu Item fix

### PR DESCRIPTION
**Pull request by ClementPadovani**
This fixes the following issue:

https://github.com/supermarin/Alcatraz/issues/340

### CODE REVIEW

**Comment by jurre** [comment]
Thanks @ClementPadovani :)

@supermarin LGTM?

File: Alcatraz/Alcatraz.m [hunk 1]
```diff
@@ -54,13 +54,25 @@ - (id)initWithBundle:(NSBundle *)plugin {
...
+										    selector: @selector(applicationDidFinishLaunching:)
+										    name: NSApplicationDidFinishLaunchingNotification
+										  object: nil];
         }];
         [self updateAlcatraz];
     }
     return self;
 }
 
+- (void) applicationDidFinishLaunching: (NSNotification *) notification
```
Comment (supermarin) [inline Alcatraz/Alcatraz.m]: Can you please rename this to something like `xcodeDidFinishLaunching`?

**Comment by supermarin** [comment]
Thanks for the contribution @ClementPadovani!

Looks good, except for the comment and it seems like tabs are used instead of spaces.
Do you mind converting the indentation to spaces like the rest of the codebase?

**Comment by ClementPadovani** [comment]
@supermarin You’re welcome. Updated the code!

**Comment by ClementPadovani** [comment]
@dmiedema Thanks!

**Comment by wczekalski** [comment]
A similar solution didn't work for me in XVim in 6.4 and had to be replaced with [this](https://github.com/XVimProject/XVim/pull/821).

**Comment by supermarin** [comment]
@ClementPadovani sorry for the delay on this.

I looked on it a few times, and it seems like tabs are still used instead of spaces:
screen shot 2015-11-02 at 12 54 40 pm

If you wouldn't mind correcting them, that'd be awesome and we're ready to merge.

**Comment by ClementPadovani** [comment]
@supermarin Hey Marin, weird I thought I had replaced those. I just replaced the tabs with spaces in this commit: https://github.com/ClementPadovani/Alcatraz/commit/5c381242628b1279aca4f85afa07ea5e69625016

**Comment by supermarin** [comment]
@ClementPadovani thanks for addressing!

Now just one more time, the upper method :)
screen shot 2015-11-03 at 9 46 14 am

**Comment by ClementPadovani** [comment]
@supermarin Sorry for missing the last one, should have fixed.

**Comment by supermarin** [comment]
@ClementPadovani thanks for the contribution! :beers:

**Pull request merged**